config/config.go

```diff
@@ -17,19 +17,11 @@ import (
 	"github.com/getsops/sops/v3/gcpkms"
 	"github.com/getsops/sops/v3/hcvault"
 	"github.com/getsops/sops/v3/kms"
-	"github.com/getsops/sops/v3/logging"
 	"github.com/getsops/sops/v3/pgp"
 	"github.com/getsops/sops/v3/publish"
-	"github.com/sirupsen/logrus"
 	"gopkg.in/yaml.v3"
 )
 
-var log *logrus.Logger
-
-func init() {
-	log = logging.NewLogger("CONFIG")
-}
```

Comment:
I got a warning from `go-staticcheck` and I could not see this being used anywhere so I removed it to avoid the warning.